### ISSUE TITLE
How to assign labels in training and test data set?

### ISSUE THREAD

**Issue by jwkimusa** [deeplearning4j/deeplearning4j#3232]
I am a novice in deeplearning4j.
I  have "Positive" and "Negative" directories in Training and Test data sets,
I used the following codes to read the data set.
 "Positive" class was assigned 0 ([1,0]) and "Negative" class was assigned 1 ([0,1]) in Training set.
However, "Positive" class was assigned 1 ([0,1] )and "Negative" class was assigned 0([1.0]) in Test set.
How can I assign 0 ([1,0])for "Positive" and 1([0,1]) for "Negative" in the two data sets?

---------------------------------------------
String labeledPathTrain = "c:\\\\Data\\Training";
String labeledPathTest  = "c:\\\\Data\Test";
 
File parentDirTrain = new File(labeledPathTrain);
File parentDirTest  = new File(labeledPathTest);

FileSplit filesInDirTrain = new FileSplit(parentDirTrain, allowedExtensions, randNumGen);
FileSplit filesInDirTest = new FileSplit(parentDirTest, allowedExtensions, randNumGen);

ParentPathLabelGenerator labelMaker = new ParentPathLabelGenerator();

ImageRecordReader recordReader = new ImageRecordReader(nHeight, nWidth, nChannels, labelMaker);
 ImageRecordReader recordReaderTest = new ImageRecordReader(nHeight, nWidth, nChannels, labelMaker);
---------------------------------------------

**Comment by crockpotveggies** [comment]
Likely duplicate of https://github.com/deeplearning4j/DataVec/issues/222
